leakcanary-android/src/main/java/com/squareup/leakcanary/internal/DisplayLeakActivity.java

```diff
@@ -216,7 +216,19 @@ void shareHeapDump() {
     startActivity(Intent.createChooser(intent, getString(R.string.leak_canary_share_with)));
   }
 
-  void updateUi() {
+  private void invalidateOptionsMenuIfAvailable() {
```

Comment:
The other one is ready, but this one I can't do. It causes a clash with the superclass's one because it is defined with `private` scope and Activity defines it with `public`. Do you want me to change it to `public` or use another name like `safeInvalidateOptionsMenu` or something?